tests/cloud/high_level_testing.json

```diff
@@ -14,5 +14,7 @@
     "mxfp6" : [1],
     "mxint8" : [1],
     "device_group" : [null],
-    "full_batch_size" : [null,3]
+    "full_batch_size" : [null,3],
+    "enable_qnn" : [false, true],
+    "qnn_config" : [null, "QEfficient/compile/qnn_config.json"]

```

Comment:
can we separate these tests from normal tests, so that we can run them nightly/weekly using marker `qnn`?

Reply:
Separating this test cases will end up with many duplication in the test code, also I think its better to have atleast some unit tests being run continuously for QNN as well to keep it updated with latest QEfficient changes. We can discuss more on this if it increases the unit test run time significantly